pkg/cmd/pulumi/util.go

```diff
@@ -1165,6 +1174,34 @@ func promptUser(msg string, options []string, defaultOption string, colorization
 	return response
 }
 
+func promptUserMultiSkippable(yes bool, msg string, options []string, defaultOptions []string,
+	colorization colors.Colorization,
+) []string {
+	if yes {
+		return defaultOptions
+	}
+	return promptUserMulti(msg, options, defaultOptions, colorization)
+}
+
+func promptUserMulti(msg string, options []string, defaultOptions []string, colorization colors.Colorization) []string {
```

Comment:
I think if we're going to extend the API for building prompts etc. we should maybe document these functions a little to help encourage their adoption.

Reply:
well code across the CLI is using survey directly, I am adding this to avoid dupplicating this code further and make it easier to migrate off survey in the future (as it is deprecated). At the same time, it is following a similar interface as the rest of the prompt functions here, but I will drop some doc comments

Reply:
done

Reply:
Imo we should eventually refactor these into the display code, and move things towards these helper functions, but that should be a separate PR.